spec/features/admin_edits_item_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.feature "Admin edits item" do
+  scenario "sees editted properties on the page" do
+    # As an admin
+    # When I visit "admin/items"
+    # And I click "Edit"
+    # Then my current path should be "/admin/items/:ITEM_ID/edit"
+    # And I should be able to upate title, description, image, and status
+    admin = create(:user, role: 1)
+    login_user(admin)
+    item = create(:item)
```

Comment:
I have fixed it, we should be covering an edge case that I initially missed, but it is now done. Thank you Rapha Barbo